itest/rust/src/plane_test.rs

```diff
@@ -0,0 +1,166 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use std::fmt::Debug;
+
+use crate::itest;
+use godot::{
+    prelude::{inner::InnerPlane, real, Plane, RealConv, ToVariant, Vector3},
+    private::class_macros::assert_eq_approx,
+};
```

Comment:
Small nitpick, please use separate lines for `use` statements without `{ }` nesting.

Also, would be nice to import symbols from their original module, not the prelude.